api/storage/elasticsearch/date.go

```diff
@@ -0,0 +1,11 @@
+package elasticsearch
+
+import "time"
+
+// startTomorrowDate returns the starte tomorrow date.
+// Example: dt = 1.1.2020 10:00:00
+// return : 1.2.2020 00:00:00
+func startTomorrowDate(dt time.Time, zone *time.Location) time.Time {
```

Comment:
I'd change the function name to something like getDayAfterDate
as it doesn't really start something and it's not really tomorrow if you provide an old time.Time argument

Reply:
thanks, function name was changed